tests/rust/test_repo.py

```diff
@@ -12,7 +31,40 @@ def test_default_repo_render(tmp_path):
         output_dir=tmp_path,
     )
 
-    actual = DiffDir(tmp_path)
-    expected = DiffDir(TEMPLATE_ROOT / "tests/rust/repo/expecteds/default")
+    actual = DiffPath(tmp_path / "repository")
+    expected = DiffPath(DEFAULT_REPO_DIR)
+
+    assert actual == expected
+
+
+def test_agpl(tmp_path):
```

Comment:
I think this is the test case that decides yay or nay on this approach to testing.
i.e. is it too codey, that it's not clear what's happening and what's being checked. Since I wrote it I know what it's doing :D